core/src/main/scala/cats/data/IndexedStateT.scala

```diff
@@ -299,6 +304,11 @@ private[data] abstract class StateFunctions {
    */
   def pure[S, A](a: A): State[S, A] = State(s => (s, a))
 
+  /**
+    * Return `A`'s empty monoid value and maintain the input state.
+    */
+  def empty[S, A: Monoid]: State[S, A] = pure(Monoid[A].empty)
```

Comment:
Should this use the same approach used above for getting the `empty` of the `Monoid`?